index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+<link rel = "stylesheet" href = "style.css" />
+<title>Portfolio</title>
+</head>
+
+<body>
+    
+<div id = "content">
+   
+    <header>
+        
+        <nav id = "menu">
+            <h4 class = "name">Ivan Gaona</h4>
+            <img src = "images/Union.png" alt = "" height = "25">
+        </nav>    
+
+    </header>
```

Comment:
- [ ] This is great to use the `<header>` as the container for the toolbar, however, there are a couple of things we need to fix for this header.
- [ ] The logo should be an anchor `<a>` tag instead of the `<h4>` tag because it is supposed to be clickable.
- [ ] The logo shouldn't stay in the `<nav>` tag since it is not part of nav links. for now, only the hamburger should stay in the `<nav>` tag
- [ ] The hamburger actually is a button, because we will need to click on it in order to show the links, so please let us use either an anchor `<a>` or a `<button>` 😉